front/src/applications/stdcm/components/StdcmForm/StdcmInputVia.tsx

```diff
@@ -45,7 +45,7 @@ const StdcmInputVia = ({ stopType, stopDuration, updatePathStepStopTime }: Stdcm
 
   return (
     stopType !== StdcmStopTypes.PASSAGE_TIME && (
-      <div className="stdcm-v2-via-stop-for stop-time">
+      <div className="stdcm-via-stop-for stop-time">
```

Comment:
I'm not sure that this class is useful, I can't find it in the css
```suggestion
      <div className="stop-time">
```